tensorflow_addons/custom_ops/activations/cc/kernels/hardshrink_op.cc

```diff
@@ -0,0 +1,77 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#define EIGEN_USE_THREADS
+
+#include "tensorflow_addons/custom_ops/activations/cc/kernels/hardshrink_op.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/register_types.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+
+namespace tensorflow {
+
+using CPUDevice = Eigen::ThreadPoolDevice;
+
+#define REGISTER_HARDSHRINK_KERNELS(type)                                  \
+  REGISTER_KERNEL_BUILDER(                                                 \
+      Name("Hardshrink").Device(DEVICE_CPU).TypeConstraint<type>("T"),     \
```

Comment:
Think we could change this to `Addons>Hardshrink`  per https://github.com/tensorflow/community/pull/126 ? It should work on the nightly and RC I believe.

Reply:
I wonder how to do this... I have tried to modify `cc/kernels/hardshrink_op.cc` and `cc/ops/hardshrink_op.cc`, but none of them gives the answer. I think I could just follow @tomerk's PR though.